docs/supported_ops.md

```diff
@@ -11569,7 +11569,7 @@ are limited.
 <td> </td>
 <td> </td>
 <td> </td>
-<td><b>NS</b></td>
+<td><em>PS<br/>UTC is only supported TZ for child TIMESTAMP</em></td>
```

Comment:
The changes in this file look suspicious. Can you merge upstream and run mvn verify again?

Reply:
Re-ran `mvn verify -DskipTests` after merging upstream branch-23.02, but I don't see any changes.

Reply:
This looks good to me. We automatically insert this in for any nested type that could support a timestamp because we don't have time zone support yet....